server/src/test/java/io/spine/server/command/model/given/reaction/TestCommandReactor.java

```diff
@@ -26,14 +26,15 @@
 
 package io.spine.server.command.model.given.reaction;
 
+import io.spine.server.command.model.CommandingReaction;
 import io.spine.server.event.EventReceiver;
 import io.spine.testing.server.model.ModelTests;
 
 import java.lang.reflect.Method;
 
 /**
  * Abstract base for test environment classes for testing
- * {@link io.spine.server.command.model.CommandReactionMethod}.
+ * {@link CommandingReaction}.
  *
  * <p>Derived classes must declare a method called {@linkplain #METHOD_NAME commandOn} so that
```

Comment:
Please review whether `TestCommandReactor` still fits the idea of renaming. Honestly, I am no longer sure whether it should remain named like this.

Reply:
Checked: it does. It's an abstract base for classes that declare `CommandingReaction` methods. I've updated documentation to make it a bit more clear.